install_client.md

```diff
@@ -1,24 +1,11 @@
 ## Install maprdb python client from internal repo ##
 
-1. Now it builds only for python 2.7 or higher. Check your python version
+1. MapR-DB python client supports python version 2.7 or higher. Check your python version
     python --version
 2. Check that you have installed pip by running "pip -V".
-If you have not installed pip, see https://packaging.python.org/guides/installing-using-linux-tools/.
+If you have not installed pip, see [instructions](https://packaging.python.org/guides/installing-using-linux-tools/).
 For example, to install pip on CentOS, run: `sudo yum install python-pip`
-3. Create a `pip.conf`. Config file path depends from your OS, you can find instruction [here](https://pip.pypa.io/en/stable/user_guide/#configuration). For centos its `/etc/pip.conf`
-4. Modify the `pip.conf` file.
-* **CV repo**
-    ```
-    [global]
-    extra-index-url = http://cv:Artifactory4CV!@34.197.167.22/artifactory/api/pypi/pypi-mapr-local/simple
-    trusted-host = 34.197.167.22
-    ```
-* **MapR repo**
-    ```[global]
-    extra-index-url = http://artifactory.devops.lab/artifactory/list/pypi-mapr-local/simple
-    trusted-host = artifactory.devops.lab
-    ``````
+3. Execute in terminal `pip install maprdb-python-client`
 
-5. Execute in terminal `pip install maprdb-python-client`
-
-Now you can use MapR-DB python client as a library.
\ No newline at end of file
+Now you can use MapR-DB python client as a library.
+More details and usage examples you can find in the [MapR official documentation](https://mapr.com/docs/61/MapR-DB/JSON_DB/GettingStartedPythonOJAI.html).
```

Comment:
Please, change to: You can find more examples for using the MapR-DB Python client [here](https://mapr.com/docs/61/MapR-DB/JSON_DB/GettingStartedPythonOJAI.html).

Reply:
Changed